drainer/checkpoint/checkpoint.go

```diff
@@ -19,6 +19,14 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	// StatusNormal means server quit normally, data <= ts is synced to downstream
+	StatusNormal int = 0
```

Comment:
Worth a better name, like `StatusConsistent` and `StatusInconsistent`

Reply:
Good Change!

Reply:
rename as `StatusConsistent` https://github.com/pingcap/tidb-binlog/pull/883/commits/002d4493a4f5cfcb06bd41acc762788271b5c2d9